### DISCUSSION TITLE
Unsaved input in otomatis dev mode

### DISCUSSION THREAD

**Discussion by hanifaudah** [bahasa-ai/otomatis-guidance#3] [General]
Sometimes input is not saved when modifying data in an application’s authorization form because saving only happens on the 5th step. This makes debugging a bit difficult because this process requires a bit of trial & error.
![image](https://user-images.githubusercontent.com/56548631/128158737-77e589b3-388e-4359-8407-20787060cf72.png)
It would be much more convenient if there is an option to save data on each page :)

**Comment by mgilangjanuar** [comment]
Thank you for the report, this issue has been solved in July 30th

https://user-images.githubusercontent.com/62420151/127609748-fe6b9ff8-e57f-4f84-b046-1fc7e9061296.mp4
